guides/advanced_guides/binary.md

```diff
@@ -1,86 +1,114 @@
-# MeiliSearch Binary
+# Installation
 
-## Downloading from Curl
+## Download and launch
+:::: tabs
 
-This script will download the **latest stable release** of MeiliSearch.
+::: tab cURL
+Download the **latest stable release** of MeiliSearch with **curl**.
+
+Launch the MeiliSearch package to start MeiliSearch's server.
```

Comment:
```suggestion
Launch the MeiliSearch binary to start MeiliSearch's server.
```

Reply:
I've just seen you changed all the "binary" by "package". But, for me, we "download a package" but we never "launch a package", we "launch a binary".
But maybe you already discuss that terminology with Kero or Thomas and I'm wrong about that.